### PR TITLE
fix: remove blue banner on specific pages

### PR DESCRIPTION
**Pull request by Holybasil**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204422365293939

### CODE REVIEW

File: src/components/Header/announcement.tsx [hunk 1]
```diff
@@ -20,6 +22,10 @@ const ReadMoreLink = styled("a")(
 )
 
 const Announcement = () => {
+  const { pathname } = useLocation()
+  if (pathname.includes("/alpha")) {
```
Comment (OrestTa) [inline src/components/Header/announcement.tsx:26]: can we be more specific? this excludes e.g. https://deploy-preview-330--scroll-io.netlify.app/blog/alphaTestnet

File: src/components/Header/announcement.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
+import { useLocation } from "react-router-dom"
+
 import { Stack } from "@mui/material"
 import { styled } from "@mui/system"
 
 const AnnouncementStack = styled(Stack)(
   ({ theme }) => `
     line-height: 2.6rem;
-    background: #00A6F2;
+    background: ${theme.palette.link.main};
```
Comment (OrestTa) [inline src/components/Header/announcement.tsx:9]: good proactive thinking, thanks! :)

**Review by OrestTa — CHANGES_REQUESTED** [review]

**Comment by Holybasil** [comment]
It 

> btw @Holybasil note that we will probably be moving towards a single master navbar and a more flat URL structure to showcase the network features more (e.g. https://scroll.io/alpha/bridge -> https://scroll.io/bridge ) soon; cc @szgbo

what is the ETA? Also, are there detailed routing mapping rules between old path and new path?

**Comment by OrestTa** [comment]
> It
> 
> > btw @Holybasil note that we will probably be moving towards a single master navbar and a more flat URL structure to showcase the network features more (e.g. https://scroll.io/alpha/bridge -> https://scroll.io/bridge ) soon; cc @szgbo
> 
> what is the ETA? Also, are there detailed routing mapping rules between old path and new path?

May 4, cf. https://scrollco.slack.com/archives/C03S72JTBGC/p1682631287030189 (incl. figma -> probably only one-level/flat URLs) & https://app.asana.com/0/0/1204307679338092/f

**Review by OrestTa — APPROVED** [review]

**Pull request merged**